.github/workflows/bazel.yml

```diff
@@ -0,0 +1,106 @@
+name: Bazel
+
+on:
+  workflow_call:
+    inputs:
+      name:
+        description: Name of workflow
+        required: false
```

Comment:
What happens when the name is not passed?

Reply:
It would use the default name of the workflow: `bazel`.